src/main/kotlin/com/arkivanov/composnake/Main.kt

```diff
@@ -79,9 +91,13 @@ private fun Board(board: Board) {
         return
     }
 
-    Column {
+    Column(
+        modifier = Modifier.padding(COL_PADDING)
```

Comment:
Why do we need paddings now? Could you please attach a screenshot?

Reply:
With padding: 
![image](https://user-images.githubusercontent.com/661056/184549496-20adab08-ec54-45b1-b2d4-fb4b6d2c6027.png)

Without padding:
![image](https://user-images.githubusercontent.com/661056/184549515-602264bb-ca39-4a1e-a76a-4c0efea0198a.png)

I see that I should make it more to match your version. Here it is with 32.dp padding:

![image](https://user-images.githubusercontent.com/661056/184549700-6826cd3c-3b26-429c-8116-f4041adb1a3c.png)

I notice it displays the title less prominently (at least on my Windows 11 desktop).